src/friktion/infos.ts

```diff
@@ -176,6 +201,28 @@ infos = class InstanceFriktion {
       ])
     );
   }
+  static async getAllEntropyMetadataSet(connection: Connection) {
+    let memcmpFilter: MemcmpFilter = { memcmp: { offset: 0, bytes: "S4L1u3M2boH" } };
+    let config: GetProgramAccountsConfig = { filters: [memcmpFilter] };
+    let allExtraDataKeys = await connection.getProgramAccounts(VOLT_PROGRAM_ID, config);
+    return new Map(
+      allExtraDataKeys.map((key) => [
+        key.pubkey.toString(),
+        { ...ENTROPY_METADATA_LAYOUT.decode(key.account.data) } as types.EntropyMetadata,
+      ])
+    );
+  }
+  static async getAllEpochInfoSet(connection: Connection) {
```

Comment:
Make the function private and add `_` to the function name of `getAllRoundSet`, `getAllExtraDataSet`, `getAllEntropyMetadataSet`, `getAllEpochInfoSet`